packages/ts-scripts/src/vite/base-vite.js

```diff
@@ -92,7 +92,7 @@ module.exports = (config, appName) =>
       alias: [{ find: '@', replacement: `${path.resolve(process.cwd(), 'src')}/` }],
     },
     server: {
-      host: true,
+      host: 'localhost',
```

Comment:
Not sure about this change? Did you have an issue with the config?

Reply:
I did indeed. I had to change this so that it would run in a windows WSL environment.